dask/dataframe/core.py

```diff
@@ -4825,10 +4810,6 @@ def apply(
         axis = self._validate_axis(axis)
         pandas_kwargs = {"axis": axis, "raw": raw, "result_type": result_type}
 
-        if not PANDAS_GT_100:
-            pandas_kwargs["broadcast"] = broadcast
-            pandas_kwargs["reduce"] = None
-
```

Comment:
broadcast is not used anywhere anymore and it should be deprecated

Reply:
Ok pushed some code with a warning. PTAL